vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/treegrid/TreeGrid.java

```diff
@@ -1056,4 +1056,33 @@ public HierarchicalDataProvider<T, SerializablePredicate<T>> getDataProvider() {
     public void scrollToIndex(int rowIndex) {
         super.scrollToIndex(rowIndex);
     }
+
+    /**
+     * Scrolls to the row with the given indexes. The indexes are hierarchical,
+     * starting with the root index.
```

Comment:
Why not use the [description](https://github.com/vaadin/web-components/blob/45cdb78cac837227eafb3fa4a750a5ae632ed704/packages/grid/src/vaadin-grid-data-provider-mixin.js#L543-L551) from the web component API?

Reply:
Addressed in the latest commits.